f/filebeat/filebeat_UBI_8.10.sh

```diff
@@ -0,0 +1,108 @@
+#
+# ----------------------------------------------------------------------------
+
+set -e 
+PACKAGE_NAME=filebeat
+PACKAGE_VERSION=${1:-v8.15.2}
+PACKAGE_URL=https://github.com/elastic/beats
+
+# Install required dependencies
+yum install -y git curl make wget tar gcc gcc-c++ libpcap-devel openssl openssl-devel which acl zlib-devel patch systemd-devel libjpeg-devel python3.11 python3.11-devel bzip2-devel gdbm-devel libdb-devel libffi-devel libuuid-devel ncurses-devel sqlite-devel xz xz-devel
+
+#Installing pip
+wget --no-check-certificate -q https://bootstrap.pypa.io/get-pip.py
+python3 get-pip.py
+rm get-pip.py 
+
+pip3 install wheel -v
+pip3 install "cython<3.0.0" pyyaml==5.4.1 --no-build-isolation -v
+
+# Install Rust
+export RUST_VERSION="1.76.0"
+printf -- 'Installing Rust \n'
+wget -q -O rustup-init.sh https://sh.rustup.rs
+bash rustup-init.sh -y
+export PATH=$PATH:$HOME/.cargo/bin
+rustup toolchain install ${RUST_VERSION}
+rustup default ${RUST_VERSION}
+rustc --version | grep "${RUST_VERSION}"
+
+# Install go
+export GO_VERSION=${GO_VERSION:-1.22.1}
+export GOROOT=${GOROOT:-"/usr/local/go"}
+export GOPATH=${GOPATH:-$HOME/go}
+export PATH=$PATH:$GOROOT/bin:$GOPATH/bin:/usr/local/bin
+wget https://golang.org/dl/go${GO_VERSION}.linux-ppc64le.tar.gz
+tar -C /usr/local -xvzf go${GO_VERSION}.linux-ppc64le.tar.gz
+rm -rf go${GO_VERSION}.linux-ppc64le.tar.gz
+export CGO_ENABLED="0"
+
+git clone $PACKAGE_URL
+cd beats/$PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+export GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=true
+export PYTHON_EXE=python3
+export PYTHON_ENV=/tmp/venv3
+
+if ! make ; then
+    echo "------------------$PACKAGE_NAME::Build_Fails---------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Build_Fails"
+    exit 1
+fi
+
+if ! make update ; then
+    echo "------------------$PACKAGE_NAME::Update_Fails---------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Update_Fails"
+    exit 1
+fi
+
+if ! make fmt ; then
+    echo "------------------$PACKAGE_NAME::Formatting_Fails---------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub | Fail |  Formatting_Fails"
+    exit 1
+fi
+
+if ! make unit ; then
+    echo "------------------$PACKAGE_NAME::Unit_Test_Fails-------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub  | Fail|  Build_Success_but_Unit_Test_Fails"
+    exit 2
+else
+    echo "------------------$PACKAGE_NAME::Build_and_Test_success-------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | $OS_NAME | GitHub  | Pass |  Both_Build_and_Unit_Test_Success"
+    exit 0
```

Comment:
If you exit 0 after this the system tests wont be executed. Remove this if system tests need to be executed.